Add message for no available appointments

evilstreak opened the pull request:
The API will try to find some appointments available in the next 4 weeks - if it can't, it will return an empty list. For that case we want to show the resident a useful page, rather than asking them to pick an appointment from a list of none.

Hopefully this page will never appear in LIVE, as there should always be some available appointments within the month, but if it does it needs to be useful for the resident using it.

The page looks like this:

<img width="902" alt="screen shot 2018-02-23 at 15 30 10" src="https://user-images.githubusercontent.com/74812/36602078-f21f557e-18ae-11e8-9096-4672cd2b66f5.png">

evilstreak committed:
Add message for no available appointments

The API will try to find some appointments available in the next 4
weeks - if it can't, it will return an empty list. For that case we want
to show the resident a useful page, rather than asking them to pick an
appointment from a list of none.

leeky commented on app/views/appointments/none_available.html.haml:
```diff
@@ -0,0 +1,22 @@
+      and quote your reference number
+      = succeed "." do
+        %strong= @work_order_reference
+      We're open Monday to Friday, 8am to midnight and Saturday, 9am to 1pm.
```
We added the opening hours to the i18n file, because it changed a few times during the project, i.e:

```haml
%p
  We're open
  = succeed '.' do
    = t('rcc_opening_hours')
```

Probably worth doing this here as well.

evilstreak replied:
Oh, interesting. I copied these from the `_contact_phone_numbers` partial, so I should probably update that too!

evilstreak replied:
Done.

leeky reviewed (approved):
One comment about opening hours, otherwise LGTM :shipit:

evilstreak committed:
Pull opening hours from i18n file on error pages

This is the only place in the application that's still got hardcoded
values rather than using the i18n value. Updating it should prevent this
getting missed if and when the opening hours need to be updated.
